### PR TITLE
Move the cloud output to the new Output interface

### PR DESCRIPTION
**Pull request by na--**
Follow up to https://github.com/loadimpact/k6/pull/1869 and https://github.com/loadimpact/k6/pull/1870. Closes https://github.com/loadimpact/k6/issues/1258 and moves the cloud output to the new `Output` interface.

### CODE REVIEW

File: cloudapi/util.go [hunk 1]
```diff
@@ -22,10 +22,7 @@ package cloudapi
...
 import "fmt"
 
+// URLForResults returns the cloud URL with the test run results.
 func URLForResults(refID string, config Config) string {
-	path := "runs"
-	if config.Token.String == "" {
-		path = "anonymous"
-	}
-	return fmt.Sprintf("%s/%s/%s", config.WebAppURL.String, path, refID)
+	return fmt.Sprintf("%s/runs/%s", config.WebAppURL.String, refID)
```
Comment (MStoykov) [inline cloudapi/util.go:27]: nit ... maybe just concat them with `+` ...?

**Review by MStoykov — APPROVED** [review]
LGTM.

**Review by MStoykov — CHANGES_REQUESTED** [review]
actually just checked 2 or 3 things I wasn't sure about with the config and a thing that definitely doesn't work is picking up my subscription from the config file

File: cloudapi/util.go (hunk 1 above)
Comment (na--) [reply cloudapi/util.go]: resolved by https://github.com/loadimpact/k6/compare/a67b3a127823480eff7038aa32f812d93ad5a2ac..7d50a9ea51f50006cad35a246b29ac1623ad8699

**Review by MStoykov — APPROVED** [review]

**Comment by na--** [comment]
This is now ready for merging from my side, I've renamed all `Collector` / `c` mentions to `Output` / `out`, and I've cleaned up some of the linter issues that were present. I suggest reading the changes commit by commit.

**Review by MStoykov — APPROVED** [review]

**Pull request merged**